### PR TITLE
#164653002 Change Os.environ to Config

### PR DESCRIPTION
**Pull request by BrianSerem**
**Description**

This covers the proposed change of switching how we retrieve environment variables. Currently, some parts of the codebase use config('VARIABLE_NAME') while some other parts use os.environ['VARIABLE'].
<hr>

**Type of change**

- [x] new chore (non-breaking change).

**Change log:**
From now on, environment variables will be reference as config('VARIABLE_NAME') instead of os.environ['VARIABLE_NAME']

<hr>

**How has this been tested**

- [x] End to End
- [x] Integration
<hr>

**Checklist:**

- [x] This follows all the guidelines for this project.
- [x] This serves as an update to the team on proposed changes.
- [x] Test coverage remained unchanged.

**Related Stories**
<hr>

[#164653002](https://www.pivotaltracker.com/n/projects/2313280/stories/164653002)

### CODE REVIEW

**Review by Etomovich — APPROVED** [review]

**Review by RachelleMaina — APPROVED** [review]

**Review by ElMonstro — APPROVED** [review]
Approved

**Review by MandelaK — APPROVED** [review]

**Review by cob04 — APPROVED** [review]
good

File: requirements.txt [hunk 1]
```diff
@@ -30,20 +30,23 @@ lazy-object-proxy==1.3.1
...
 pylint==2.3.0
 python-decouple==3.1
+python-http-client==3.1.0
 python-social-auth==0.3.6
 python3-openid==3.1.0
 pytz==2018.9
 requests==2.21.0
 requests-oauthlib==1.2.0
 ruamel.yaml==0.15.89
+sendgrid==5.6.0
```
Comment (abulojoshua1) [inline requirements.txt:55]: @BrianSerem, does this dependency contribute to this PR?

File: requirements.txt [hunk 2]
```diff
@@ -30,20 +30,23 @@ lazy-object-proxy==1.3.1
 MarkupSafe==1.1.1
 mccabe==0.6.1
 mock==2.0.0
-pbr==5.1.3
 oauthlib==3.0.1
+pbr==5.1.3
+psycopg2==2.7.7
```
Comment (abulojoshua1) [inline requirements.txt:39]: @BrianSerem, does this dependency contribute to this PR?

File: requirements.txt [hunk 3]
```diff
@@ -30,20 +30,23 @@ lazy-object-proxy==1.3.1
...
 psycopg2-binary==2.7.7
 pycodestyle==2.5.0
 pyflakes==2.1.0
 PyJWT==1.4.2
 pylint==2.3.0
 python-decouple==3.1
+python-http-client==3.1.0
```
Comment (abulojoshua1) [inline requirements.txt:47]: @BrianSerem, does this dependency contribute to this PR?

File: requirements.txt (hunk 1 above)
Comment (BrianSerem) [reply requirements.txt:55]: I hope this has been resolved

File: requirements.txt (hunk 2 above)
Comment (BrianSerem) [reply requirements.txt:39]: Yes, it was inside one of the packages as a dependency, it came up after running `pip freeze > requirements.txt`

Comment (abulojoshua1) [reply requirements.txt:39]: Do we need it too?

**Review by abulojoshua1 — APPROVED** [review]
LGTM

**Comment by BrianSerem** [comment]
Read instructions carefully

**Pull request merged**